src/locale.en.js

```diff
@@ -33,23 +34,37 @@ export const campsite = {
         accessibility: 'Accessibility',
         facilities: 'Facilities',
         price: 'Price',
+        features: 'Features',
         price_options: {
             free: 'Free',
             paid: 'Paid',
+            paid_reservable: 'Paid & Reservable',
             permit: 'Permit'
         },
         facilities_options: {
-            full_service: 'Full Service',
-            some_facilities: 'Some',
-            no_facilities: 'None'
+            restrooms: 'Restrooms',
+            potable_water: 'Potable Water',
+            showers: 'Showers',
+            metal_firepit: 'Metal Fire Pit',
+            tables: 'Tables',
+            none: 'None'
         },
         accessibility_options: {
             paved_road: "Paved Road",
             dirt_road: "Dirt Road",
             uneven_terrain: "Uneven Terrain",
-            fourbyfour: "4x4(recommended)",
-            fourbyfour_clearence: "4X4 Drive High Clearance",
+            fourbyfour: "4x4 Recommended",
+            fourbyfour_clearence: "4X4 Required",
             hike_access: "Hike Access"
+        },
+        features_options: {
+            evergreens: 'Evergreens',
+            color_changing_trees: 'Aspen/Color-Changing Trees',
+            lake_view: 'Lake View',
+            river_view: 'River/Creek View',
+            secluded: 'Secluded',
+            mountain_views: 'Mountain Views',
+            easy_access: 'Easy Access'
```

Comment:
Added `features`, changed the options in `facilities` to be a list to choose from, and changed the wording slightly in `accessibility`.